CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v.2.17.0 - 2020-06-17
```

Comment:
don't edit the changelog. You don't know when the new version will be published

Reply:
okey